Automattic/sensei
# Add common question block settings

### Changes proposed in this Pull Request

* Add settings to the question blocks, stored in the `options` attribute
* All question blocks: **Grade**
* True/false, multiple choice, gap fill: **Answer Feedback**
* Single line, multi line, file upload: **Grading Notes** (formerly Guide/Teacher Notes)
* Multi choice: **Random order**
* Add panel for question categories (not implemented in this PR)

### Testing instructions

* Add a quiz to the lesson with all type of questions
* Change the various settings from the sidebar
* Make sure they are persisted when saving the post. (In the post, this is not tied to the REST API)


### Screenshot / Video

<img width="280" alt="image" src="https://user-images.githubusercontent.com/176949/107983451-131ff000-6fc6-11eb-93bc-89ed94ff237d.png">
